Demo/EarlGreyExample/README.md

```diff
@@ -11,7 +11,7 @@ different functionalities provide by EarlGrey, written in Objective-C and Swift.
 
 ### CocoaPods
 
-Execute `gem install earlgrey` then `pod install` from the directory where you have downloaded
+Execute `gem install earlgrey` then `pod install --repo-update` from the directory where you have downloaded
 the source of the example, then `open EarlGreyExample.xcworkspace` to open the example in xcode.
```

Comment:
Won't `--repo-update` update all the repos in the spec irrespective of whether a repo is required for EarlGrey or not? Maybe have this as part of a note?

Reply:
Yes, the Example only has one pod though. We'd want that to be up to date. I'll update the readme to clarify.

Reply:
Updated. https://github.com/CocoaPods/CocoaPods/issues/5697#issuecomment-236604410 has more background.